docs/installation/linux/centos.md

```diff
@@ -139,54 +177,63 @@ makes the ownership of the Unix socket read/writable by the `docker` group.
 
 To create the `docker` group and add your user:
 
-1. Log into Centos as a user with `sudo` privileges.
+1. Log into your machine as a user with `sudo` or `root` privileges.
 
 2. Create the `docker` group.
 
-    `sudo groupadd docker`
+    ```bash
+    $ sudo groupadd docker
+    ```
 
 3. Add your user to `docker` group.
 
-    `sudo usermod -aG docker your_username`
+    ```bash
+    $ sudo usermod -aG docker your_username`
+    ```
 
 4. Log out and log back in.
 
     This ensures your user is running with the correct permissions.
 
 5. Verify your work by running `docker` without `sudo`.
 
-		$ docker run hello-world
+    ```bash
+    $ docker run hello-world
+    ```
 
 ## Start the docker daemon at boot
 
 To ensure Docker starts when you boot your system, do the following:
```

Comment:
There is a lot of "to ensure" in this content ... let's consider rewriting this to something like:

Configure the Docker daemon to start automatically when the host starts: